### PR TITLE
{numlib}[gompi/2022a] PFFT v20181230

### PR DESCRIPTION
**Pull request by lexming**

### CODE REVIEW

**Comment by lexming** [comment]
@boegelbot: please test @ generoso

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/905f5c2df3462a25665a72a4dd863c1f for a full test report.

**Review by verdurin — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/p/PFFT/PFFT-20181230-gompi-2022a.eb [hunk 1]
```diff
@@ -0,0 +1,39 @@
+easyblock = 'ConfigureMake'
+
+name = 'PFFT'
+version = '20181230'
+_commit = 'e4cfcf9902d0ef82cb49ec722040932b6b598c71'
```
[the remark this answers is not in the document]
Comment (lexming) [reply easybuild/easyconfigs/p/PFFT/PFFT-20181230-gompi-2022a.eb:5]: `local_commit` or `_commit` both are ok

[the remark this answers is not in the document]
Comment (smoors) [reply easybuild/easyconfigs/p/PFFT/PFFT-20181230-gompi-2022a.eb:5]: https://docs.easybuild.io/easyconfig-files-local-variables/?h=local#easyconfig_files_local_variables_naming_scheme

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @lexming!

**Pull request merged**